streamlit_webrtc/frontend/package.json

```diff
@@ -21,6 +21,7 @@
     "chroma-js": "^3.1.2",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
+
```

Comment:
Delete this unnecessary line. You don't need to add any changes on `package.json`.